x-pack/elastic-agent/docs/elastic-agent.asciidoc

```diff
@@ -1,78 +1,255 @@
+
+["source","sh",subs="attributes,callouts"]
+------------------------------------------------
+curl -L -O https://artifacts.elastic.co/downloads/beats/elastic-agent/elastic-agent-{version}-linux-x86_64.tar.gz
+tar xzvf elastic-agent-{version}-linux-x86_64.tar.gz
+------------------------------------------------
+
+endif::[]
+
+[[win]]
+*win:*
+
+ifeval::["{release-state}"=="unreleased"]
+
+Version {version} of {beatname_uc} has not yet been released.
+
+endif::[]
+
+ifeval::["{release-state}"!="unreleased"]
+
+. Download the Elastic Agent Windows zip file from the
+https://www.elastic.co/downloads/beats/elastic-agent[downloads page].
+
+. Extract the contents of the zip file into `C:\Program Files`.
+
+. Rename the `elastic-agent-<version>-windows` directory to `Elastic-Agent`.
+
+. Open a PowerShell prompt as an Administrator (right-click the PowerShell icon and select *Run As Administrator*).
+
+. From the PowerShell prompt, run the following commands to install Filebeat as a
+Windows service:
++
+[source,shell]
+----------------------------------------------------------------------
+PS > cd 'C:\Program Files\Elastic-Agent'
+PS C:\Program Files\Elastic-Agent> .\install-service-elastic-agent.ps1
+----------------------------------------------------------------------
+
+NOTE: If script execution is disabled on your system, you need to set the execution policy for the current session to allow the script to run. For example: `PowerShell.exe -ExecutionPolicy UnRestricted -File .\install-service-elastic-agent.ps1`.
+
+endif::[]
+
+=== Step 3: Run Elastic Agent
+
+If Elastic Agent is not installed as an auto-starting service it is possible to start in manually running.
+
+
+[source,shell]
+----------------------------------------------------------------------
+$ ./elastic-agent run
+----------------------------------------------------------------------
+
+[[elastic-agent-execution-modes]]
+== Execution modes
+
+Elastic Agent is capable of running in two modes: standalone or fleet. Difference between these is in a way how where agent is configured and managed.
+
+=== Standalone mode
+
+With standalone mode agent is configured locally and managed manually. Each agent is configured to be in this mode by default after installation.
+Elastic Agent reads configuration file provided by `-c` argument while agent is being started or uses a default configuration located in the same directory called `elastic-agent.yml`.
+
+For configuration options see link:elastic-agent_configuration_example.yml[Elastic Agent Configuration Example]
+
+=== Fleet mode
+
+With fleet mode agent is managed remotely. It uses trusted kibana instance with enabled _Ingest Manager_ and _Fleet_ to retrieve configurations from and report any agent events to.
+
+Process of creating trusted communication channel between _Elastic Agent_ and _Kibana_ is called `enrolling an agent`.
+
+To enroll an _Elastic Agent_ to _Fleet_ agent provides an `enroll` command.
+
+
```

Comment:
I think you should break this into steps because notes usually contain ancillary info. See my edits below.